From bb0431dcfefb1d710b0521d18129c96218fd0900 Mon Sep 17 00:00:00 2001
From: Zhenbo Gao <zhenbo.gao@windriver.com>
Date: Mon, 3 Sep 2018 17:37:45 +0800
Subject: [PATCH 002/176] Revert "tipc: call tipc_rcv() only if bearer is up
 in tipc_udp_recv()"

This reverts commit 8560506d5b7a86003d5905b4a0c647f2886f4ad9.

A big patch set for upgrading TIPC(to v4.18) will come soon

Signed-off-by: Zhenbo Gao <zhenbo.gao@windriver.com>
---
 net/tipc/udp_media.c |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/net/tipc/udp_media.c b/net/tipc/udp_media.c
index 3deabca..ecca64f 100644
--- a/net/tipc/udp_media.c
+++ b/net/tipc/udp_media.c
@@ -371,6 +371,10 @@ static int tipc_udp_recv(struct sock *sk, struct sk_buff *skb)
 			goto rcu_out;
 	}
 
+	tipc_rcv(sock_net(sk), skb, b);
+	rcu_read_unlock();
+	return 0;
+
 rcu_out:
 	rcu_read_unlock();
 out:
-- 
1.7.9.5

